src/OpenTelemetry.Exporter.Jaeger/README.md

```diff
@@ -45,6 +47,16 @@ See the
 [`TestJaegerExporter.cs`](../../examples/Console/TestJaegerExporter.cs)
 for an example of how to use the exporter.
 
+## Resources and Process Tags
+
+Jaeger sends information about the originating process alongside exported
```

Comment:
I am thinking - its okay to skip this whole section. These are already covered in the spec that Resource is mapped to Jaeger's Span.Process.

Reply:
removed